ivy/functional/backends/jax/general.py

```diff
@@ -93,7 +94,7 @@ def set_item(
     ret = x.at[query].set(val)
     if copy:
         return ret
-    return ivy.inplace_update(x, _to_device(ret))
```

Comment:
I think we should revert this change as it seems unrelated to our changes

Reply:
there's no reference anywhere to this function `_to_device ` so my IDE flag it as an error, that why I removed it. But I'll revert it